scripts/python/mps/scrapeLoksabhaMPs.py

```
@@ -0,0 +1,773 @@
+    params = {
+        "page": 1,
+        "limit": 570,  # increase for faster pagination
+        "search": "",
+        "sortBy": "utilizationPercentage",
+        "order": "desc",
+        "house": "Lok Sabha",
+        "ls_term": 18
+    }
+
+    constituency_to_mp_id = {}
+    count=0;
```

Ruff (0.14.4)

Statement ends with an unnecessary semicolon

Remove unnecessary semicolon

(E703)